### DISCUSSION TITLE
Problem with selecting TWAIN driver

### DISCUSSION THREAD

**Discussion by tester6gg** [cyanfish/naps2#311] [Support]
Hello. In 6.1.2 version i can set TWAIN driver in scan profiles instead of WIA (WIA adds some odd white margins). In newest 7.3.1 when i'm trying to select TWAIN i got a message after couple of seconds (screen)
![twain](https://github.com/cyanfish/naps2/assets/140113237/2966749c-175e-4f13-b5bc-4a1c4d40ffcf)
In 7.0b1 there was a change "TWAIN support has been reworked" I think it may change something and cause this problem.
I had old Brother DCP-145C multifunction printer with drivers from Brother, running on windows 11.
But I want to thank for naps2 anyway. Even on 6.1.2 release it makes scanning process much easier.

**Comment by cyanfish** [comment]
Can you follow [these instructions](https://www.naps2.com/doc/troubleshooting#error-log) to collect error logs?

You can also try uninstalling and reinstalling NAPS2.

**Reply by tester6gg** [reply]
Reinstalling fixed it. I'm ashamed ;)
It was probably because I installed print driver after I installed naps2. Thank you!

**Reply by cyanfish** [reply]
You probably used the MSI installer after previously using the EXE installer, they don't mix well.
